analyzer/codechecker_analyzer/analyzer_context.py

```diff
@@ -116,9 +119,24 @@ def __set_version(self):
             self.__package_git_tag = package_git_dirtytag
 
     def __populate_analyzers(self):
+        """ Set analyzer binaries for each registered analyzers. """
+        check_env = None
+        analyzer_from_path = is_analyzer_from_path()
+        if analyzer_from_path:
```

Comment:
Shouldn't this be:
```suggestion
        if not analyzer_from_path:
```
extend the env only if the analyzer id not from the path.